src/main.py

```diff
@@ -21,6 +23,19 @@
 import persistent_state
 import song_utils
 
+
+def setup_logging(log_level):
+    logger = logging.getLogger("nextcord")
+    logger.setLevel(log_level)
+    handler = logging.StreamHandler(stream=sys.stderr)
```

Comment:
```suggestion
    handler = logging.FileHandler(stream=sys.stderr, filename='nextcord.log', encoding='utf-8', mode='w')
```

there's some extra parameters that are recommended from the [nextcord docs](https://docs.nextcord.dev/en/stable/logging.html), I don't know if you want to maybe include these as well

Reply:
LGTM, otherwise

Reply:
yeah, totally disregard this since the logging on those docs are for writing to a file rather than streaming to a terminal

Reply:
Note in the PR I say this writes to stderr instead of a file, that's what the modification I did from the nextcord example is. I did this so we can just use systemd  to log everything for now since we still have some print logging. Later it would probably be better to move everything to the logging module and maybe then we can revisit whether io streams or a file is better.

Reply:
Oh lol you already replied. cool